packages/react-integration/demo-app-ts/src/components/demos/TopologyDemo/PipelineLayout.tsx

```diff
@@ -77,6 +78,10 @@ const getModel = (layout: string): Model => {
 export const PipelineLayout = withTopologySetup(() => {
   useLayoutFactory((type: string, graph: Graph): Layout | undefined => new PipelineDagreLayout(graph, { nodesep: 95 }));
   useComponentFactory(pipelineComponentFactory);
+  const controller = useVisualizationController();
+  controller.setFitToScreenOnLayout(true);
+  controller.setFitToScreenOnLayout(true);
```

Comment:
Is this line printed twice intentionally?

Reply:
nope, thanks!